### DISCUSSION TITLE
Invalid copilot token: missing token: 403

### DISCUSSION THREAD

**Discussion by Tythee** [community/community#112664] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

i bought the copilot，but Invalid copilot token
2024-03-16 12:04:45.556 [info] [auth] Invalid copilot token: missing token: 403 
2024-03-16 12:04:45.586 [error] [default] Extension activation failed: "Contact Support. You are currently logged in as Tythee."

**Comment by gil-son** [comment, marked as answer]
Hello, how are you? I feel for that situation

Acces the [Github Support](https://support.github.com/)

Click on My Ticket - New Ticket - Finally describe it

<hr/>

If I supported you consider to mark my answer

**Comment by HelloDuoLA** [comment]
> ### Select Topic Area
> Question
> 
> ### Body
> i bought the copilot，but Invalid copilot token 2024-03-16 12:04:45.556 [info] [auth] Invalid copilot token: missing token: 403 2024-03-16 12:04:45.586 [error] [default] Extension activation failed: "Contact Support. You are currently logged in as Tythee."

Did you solve the problem with the help of Github Support？

**Reply by Tythee** [reply]
yes

**Reply by HelloDuoLA** [reply]
Can you tell me how they told you to operate? I messaged them a few days ago, but I haven't heard back yet.

**Reply by manyuan-li** [reply]
I got the same problem, could u plz tell me how to fix it?

**Comment by Tythee** [comment]
> I got the same problem, could u plz tell me how to fix it?

Go to github support, click my tickets in the upper right corner, and then click to report bug
